### DISCUSSION TITLE
How can I use the $ .fn.dataTable.ext.errMode function

### DISCUSSION THREAD

**Discussion by zero-53** [l-lin/angular-datatables#1601] [Q&A] [answered]
My table works correctly in all functions but when a user loses privileges to the data temporarily or the server sends a 403 response, the alert appears on the screen of the ajax error! That response from the server is already handled but the table sends an alert. reading the DataTablet documentation I found that it is removed with the function $ .fn.dataTable.ext.errMode: 'none'. what I don't know is how to do it here

![image](https://user-images.githubusercontent.com/55466533/135735223-fe75d88f-c649-4969-b530-9b939bb19888.png)

**Comment by shanmukhateja** [comment, marked as answer]
Hi,

You add that line in `ngOnInit` of your table component. Just paste the line as-is and you should be good.

**Reply by zero-53** [reply]
Thanks! it works
